src/Compilers/CSharp/Portable/Symbols/NullableFlowState.cs

```diff
@@ -9,7 +9,19 @@ namespace Microsoft.CodeAnalysis.CSharp
     /// </summary>
     internal enum NullableFlowState : byte
     {
-        NotNull,
-        MaybeNull
+        /// <summary>
+        /// Not null.
+        /// </summary>
+        NotNull = 0b00,
+
+        /// <summary>
+        /// Maybe null (type is nullable).
+        /// </summary>
+        MaybeNull = 0b01,
+
+        /// <summary>
+        /// Maybe null (type may be not nullable).
+        /// </summary>
+        MaybeDefault = 0b11,
```

Comment:
Why `0b11` and not `0b10`?

Reply:
So we can (continue to) use bit-wise `|` and `&` for `Join()` and `Meet()`.

---
In reply to: [348763602](https://github.com/dotnet/roslyn/pull/39778#discussion_r348763602) [](ancestors = 348763602)